configuration/ReadMe.md

```diff
@@ -61,15 +61,15 @@ This file is currently used by older versions of the tooling to map versions for
 
 
 ## igniteVersionToDisplayName.properties
-This file maps the Fuse Ignite version to a human readable name to be displayed in the New Fuse Ignite Wizard. It replaces the old ignite.properties.
+This file maps the Fuse Online version to a human readable name to be displayed in the New Fuse Online Extension Wizard. It replaces the old ignite.properties.
 
 Syntax:
 
-	<ignite.version>=<human readable name of this version>
+	<fuse.online.version>=<human readable name of this version>
```

Comment:
the online file should be changed also? Or need to use another one to keep backward-compatibility?

Reply:
I don't get your question. That is just a key name for the readme. The config file only contains versions.

Reply:
I don't understand to what this key is referring then. Where is it used?

Reply:
It is just an explanation on how to fill in values to the config file. Its like <key>=<value> which just tries to make it easy for users to fill in the correct values.

Reply:
wow I just get it, I thought the ignite.version or fuse.online.version was used directly as a key somewhere. My bad